setup.py

```diff
@@ -323,7 +329,7 @@ def get_common_options(build_ext):
 
     if gpu_allreduce == 'NCCL':
         have_nccl = True
-        nccl_include_dirs, nccl_lib_dirs = get_nccl_dirs(
+        nccl_include_dirs, nccl_lib_dirs, nccl_lib_name = get_nccl_vals(
             build_ext, cuda_include_dirs, cuda_lib_dirs, cpp_flags)
     else:
         have_nccl = False
```

Comment:
Can you add `nccl_lib_name = None` after `nccl_include_dirs = nccl_lib_dirs = []` below to remain consistent?  Or, just remove the `nccl_include_dirs = nccl_lib_dirs = []` line.

Reply:
Or, even better, return `nccl_libs` from `get_nccl_vals()` which is an array of library names.  Then you can set all three to an empty array, and it's also a nice default value - better than an array wth None element.